docs/raising-events.md

```diff
@@ -87,6 +87,7 @@ This method may also be used from C# if you don't want to rely on late binding.
 ## Limitations
 
 The approach described above for raising events doesn't work in some situations:
+

```

Comment:
The missing blank line caused the subsequent bullet points to appear on the same line as the previous paragraph.